app/views/charities/index.html.erb

```diff
@@ -1,4 +0,0 @@
-
```

Comment:
This was deleted because it was only used for the first 'simple' search functionality. With the Sunspot Gem, Charities#Index is not longer needed. So I deleted all related files.